compiler/luci/pass/src/helpers/LayerInfoMap.test.cpp

```diff
@@ -43,6 +43,61 @@ class SoftmaxTestGraph : public luci::test::TestIOGraph
   luci::CircleSoftmax *_softmax = nullptr;
 };
 
+class SplitAddTestGraph : public luci::test::TestIOGraph
+{
+public:
+  void init(void)
+  {
+    TestIOGraph::init({6, 1, 2}, {3, 1, 2});
+    _split_dim = g()->nodes()->create<luci::CircleConst>();
+    {
+      _split_dim->rank(1);
+      _split_dim->dtype(loco::DataType::S32);
+      _split_dim->size<loco::DataType::S32>(1);
+      _split_dim->at<loco::DataType::S32>(0);
+      _split_dim->shape({1});
+      _split_dim->name("split_dim");
+    }
+
+    _split = g()->nodes()->create<luci::CircleSplit>();
+    {
+      _split->input(input());
+      _split->num_split(2);
+      _split->split_dim(_split_dim);
+      _split->name("split");
```

Comment:
Could you make a test that CircleSplit has the same name with the first CircleSplitOut ? It is the actual behavior of `luci/import`.

```suggestion
      _split->name("split0");
```